en/components/slider/slider.md

```diff
@@ -45,35 +38,6 @@ import { IgxSliderModule } from 'igniteui-angular';
 export class AppModule {}
 ```
 
-#### Continuous Slider
-Let's start with a simple continuous slider. First, specify the slider type by setting the [`continuous`]({environment:angularApiUrl}/classes/igxslidercomponent.html#continuous) input to true. Next, define the minimum and maximum values using [`minValue`]({environment:angularApiUrl}/classes/igxslidercomponent.html#minvalue) and [`maxValue`]({environment:angularApiUrl}/classes/igxslidercomponent.html#maxvalue). We also bind the slider [`value`]({environment:angularApiUrl}/classes/igxslidercomponent.html#value) to a property in our component called "volume".
-
-> [!NOTE]
-> Continuous slider doesn't have step indicators over the track and visible thumb labels during interaction.
-
-```html
-<!--sample.component.html-->
-
-<igx-slider id="slider" [minValue]="0" [maxValue]="100" [continuous]=true [(ngModel)]="volume" ></igx-slider>
-<label igxLabel for="slider">Volume: {{volume}}</label>
-```
-
-```typescript
-// sample.component.ts 
-
-// Set an initial value
-public volume = 20;
-```
-
-If the sample is configured properly, dragging the slider thumb should update the label below and the slider value should be limited between the specified minimum and maximum values:
-
-<div class="sample-container loading" style="height: 140px">
-    <iframe id="slide-sample-2-iframe" data-src='{environment:demosBaseUrl}/interactions/slider-sample-2' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
-</div>
-<div>
-    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slide-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
-</div>
-
```

Comment:
I think there's something wrong with your branch, it modifies the slider docs too.